content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+beta: true
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
+---
+
+<div class="alert alert-warning">This feature is currently in private beta. To request access, reach out to support@datadoghq.com.</div>
```

Comment:
we're going to start rolling out next week (jul 25)-  we can actually add these docs as public (not unlisted) without the private beta disclaimer 🙌